Sentry.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Sentry"
-  s.version      = "8.49.2"
+  s.version      = "8.49.3"
```

Comment:
`h`: We shouldn't have to update this manually. Only the GitHub release workflow should do that.